source/Orders.h

```diff
@@ -0,0 +1,123 @@
+/* Orders.h
+Copyright (c) 2024 by Amazinite
+
+Endless Sky is free software: you can redistribute it and/or modify it under the
+terms of the GNU General Public License as published by the Free Software
+Foundation, either version 3 of the License, or (at your option) any later version.
+
+Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+PARTICULAR PURPOSE. See the GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License along with
+this program. If not, see <https://www.gnu.org/licenses/>.
+*/
+
+#ifndef ORDERS_H_
+#define ORDERS_H_
+
+#include <bitset>
+#include <map>
+#include <memory>
+
+#include "Point.h"
+
+class Minable;
+class Ship;
+class System;
+
+
+// Orders represent commands that have been given to the player's escorts.
+class Orders {
+public:
+	enum OrderType : int {
+		HOLD_POSITION,
+		// HOLD_ACTIVE is the same command as HOLD_POSITION, but it is given when a ship
+		// actively needs to move back to the position it was holding.
+		HOLD_ACTIVE,
+		MOVE_TO,
+		KEEP_STATION,
+		GATHER,
+		ATTACK,
+		// FINISH_OFF is used to ATTACK ships that are disabled.
+		FINISH_OFF,
+		// MINE is for fleet targeting the asteroid for mining. ATTACK is used
+		// to chase and attack the asteroid.
+		MINE,
+		// HARVEST is related to MINE and is for picking up flotsam after
+		// ATTACK.
+		HARVEST,
+
+		// This must be last so it can be used for bounds checking.
+		TYPES_COUNT
+	};
+
+
+public:
+	// Make sure this matches the number of items in OrderType,
+	// or the build will fail.
+	static const int ORDER_COUNT = 9;
```

Comment:
Why can't you use `OrderType::TYPES_COUNT` instead of this variable? At the  very least, you could use that to initialize the variable.